database/sqlite/sqlite_health.c

```diff
@@ -8,7 +8,7 @@
 /* Health related SQL queries
    Creates a health log table in sqlite, one per host guid
 */
-#define SQL_CREATE_HEALTH_LOG_TABLE(guid) "CREATE TABLE IF NOT EXISTS health_log_%s(hostname text, unique_id int, alarm_id int, alarm_event_id int, config_hash_id blob, updated_by_id int, updates_id int, when_key int, duration int, non_clear_duration int, flags int, exec_run_timestamp int, delay_up_to_timestamp int, name text, chart text, family text, exec text, recipient text, source text, units text, info text, exec_code int, new_status real, old_status real, delay int, new_value double, old_value double, last_repeat int, class text, component text, type text);", guid
+#define SQL_CREATE_HEALTH_LOG_TABLE(guid) "CREATE TABLE IF NOT EXISTS health_log_%s(hostname text, unique_id int, alarm_id int, alarm_event_id int, config_hash_id blob, updated_by_id int, updates_id int, when_key int, duration int, non_clear_duration int, flags int, exec_run_timestamp int, delay_up_to_timestamp int, name text, chart text, family text, exec text, recipient text, source text, units text, info text, exec_code int, new_status real, old_status real, delay int, new_value double, old_value double, last_repeat int, class text, component text, type text, chart_context text);", guid
```

Comment:
Some of the fields in this `CREATE TABLE` are missing constraint like `NOT NULL`. I suggest to add them at least for `text` and `blob` fields.

Reply:
Agreed. But since adding them would take some testing all around, maybe better to do it in a separate PR. ?

Reply:
All right, I will accept this, because this is not the unique table that is needing this work, probably we need a PR to address database issues. I won't mark as resolved, because I do not want this to be forgotten. :handshake:

Reply:
Thank you @thiagoftsm !